src/main/resources/reranking-providers-config.yaml

```diff
@@ -0,0 +1,17 @@
+stargate:
+  jsonapi:
+    reranking:
+      providers:
+        nvidia:
+          is-default: true
+          display-name: Nvidia
+          enabled: true
+          supported-authentications:
+            NONE:
+              enabled: true
+          models:
```

Comment:
-1 There must be an Enum on the model to explain what sort of model it is, it must not just be the file the name of the file.